src/Compilers/Core/Portable/Operations/IMultiVariableDeclaration.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents multiple declared variables in a single declarator.
+    /// </summary>
+    /// <para>
+    /// Current Usage:
+    ///   (1) VB As New statements
+    ///   (2) VB multiple declarations in a single declarator
```

Comment:
>VB multiple declarations in a single declarator [](start = 14, length = 47)

I think this node should be created for any VB declarator, regardless how many variables it declares.